internal/core/software_renderer.rs

```diff
@@ -440,16 +440,17 @@ impl SoftwareRenderer {
     /// Render the window to the given frame buffer.
     ///
     /// The renderer uses a cache internally and will only render the part of the window
-    /// which are dirty. The `extra_draw_region` is an extra regin which will also
+    /// which are dirty. The `extra_draw_region` is an extra region which will also
     /// be rendered. (eg: the previous dirty region in case of double buffering)
     /// This function returns the region that was rendered.
     ///
-    /// The pixel_stride is the size, in pixel, between two line in the buffer
+    /// The pixel_stride is the size (in pixels) between two lines in the buffer.
+    /// It is equal `width` if the screen is not rotated, and `height` if the screen is rotated by 90°.
```

Comment:
Why is the pixel_stride equal to width? For example on an RGB565 screen with say 100 pixels, the pixel_stride is 200 bytes, that's not equal :)

```suggestion
    /// It is equal `width` if the screen is not rotated, and `height` if the screen is rotated by 90°.
```

Reply:
Yes but the pixel_stride is given in pixel not bytes in this particular function.
Here is a code snipped I use in my demo: `let region = renderer.render(&mut buffer, Self::DISPLAY_WIDTH as usize);`

Reply:
You're right :)